src/protobuf/appearances.proto

```
@@ -0,0 +1,310 @@
+// Copyright 2024 The Atlas Server Authors.
+// Protobuf schema for Tibia Client 12+ appearances.dat
+// Updated to match client 15.24
+
+syntax = "proto2";
+
+package atlas.protobuf.appearances;
```

Buf (1.69.0)

[error] Files with package "atlas.protobuf.appearances" must be within a directory "atlas/protobuf/appearances" relative to root but were in directory "src/protobuf".

(PACKAGE_DIRECTORY_MATCH)